app/DynamicsAdapter/DynamicsAdapter.Web/SearchAgency/AgencyRequestController.cs

```diff
@@ -57,7 +57,7 @@ public async Task<IActionResult> CreateSearchRequest(string requestId, [FromBody
                 catch (Exception ex)
                 {
                     _logger.LogError(ex.Message);
-                    return StatusCode(StatusCodes.Status500InternalServerError, new { Message = ex.Message });
+                    return StatusCode(StatusCodes.Status500InternalServerError, new { ReasonCode = ex.Message, Message = ex.InnerException?.Message });
```

Comment:
are we sending this to FMEP ? I don't think we are just checking.

Reply:
yes, we need to send this to Rejected Message. They ask for specific reason, and reason code. You can see this in Dragan's email on Set.24.

Reply:
I was meaning the exception we don't want to send exception message